### PR TITLE
Add albert resources

### PR DESCRIPTION
**Pull request by elabongaatuo**
# What does this PR do?

Fixes   #20055

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.
@stevhliu

### CODE REVIEW

File: docs/source/en/model_doc/albert.mdx [hunk 1]
```diff
@@ -56,13 +56,27 @@ Next sentence prediction is replaced by a sentence ordering prediction: in the i
...
+- A notebook on how to [Fine-tune ALBERT for sentence-pair classification](https://github.com/NadirEM/nlp-notebooks/blob/master/Fine_tune_ALBERT_sentence_pair_classification.ipynb).🌎
+- [Text classification](https://huggingface.co/docs/transformers/tasks/sequence_classification) chapter of the 🤗 Hugging Face Task Guides.
+- A blog post on [Deep Learning over Internet](https://huggingface.co/blog/collaborative-training) that uses ALBERT model.
+
+<PipelineTag pipeline="token-classification" /> 
+
+- A course material on [EncoderModel](https://huggingface.co/course/chapter1/5). 
+- A Transformers ALBERT document on [AlbertForTokenClassification](https://huggingface.co/docs/transformers/v4.24.0/en/model_doc/albert#transformers.AlbertForTokenClassification)
+
+<PipelineTag pipeline="Research" />
```
Comment (stevhliu) [inline docs/source/en/model_doc/albert.mdx:77]:
```suggestion
🧪 Research
```

File: docs/source/en/model_doc/albert.mdx [hunk 2]
```diff
@@ -56,13 +56,27 @@ Next sentence prediction is replaced by a sentence ordering prediction: in the i
...
 
-- [Text classification task guide](../tasks/sequence_classification)
-- [Token classification task guide](../tasks/token_classification)
-- [Question answering task guide](../tasks/question_answering)
-- [Masked language modeling task guide](../tasks/masked_language_modeling)
-- [Multiple choice task guide](../tasks/multiple_choice)
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with ALBERT. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+<PipelineTag pipeline="fill-mask" />
+- [`AlbertForMaskedLM`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/language-modeling#robertabertdistilbert-and-masked-language-modeling)
```
Comment (stevhliu) [inline docs/source/en/model_doc/albert.mdx:64]: Can you also add the script and notebook for `TFAlbertForMaskedLM` and `FlaxAlbertForMaskedLM`?

```suggestion
- [`AlbertForMaskedLM`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/language-modeling#robertabertdistilbert-and-masked-language-modeling) and [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/examples/language_modeling.ipynb).
```

File: docs/source/en/model_doc/albert.mdx [hunk 3]
```diff
@@ -56,13 +56,27 @@ Next sentence prediction is replaced by a sentence ordering prediction: in the i
...
+
+<PipelineTag pipeline="text-classification"/>
```
Comment (stevhliu) [inline docs/source/en/model_doc/albert.mdx:66]: Similar to the fill-mask section, can you also add the relevant script and notebook for text classification with PyTorch, TensorFlow and Flax? This same comment goes for the token classification section, and we're also missing a multiple choice and question answering section.  See the [BERT](https://github.com/huggingface/transformers/blob/003a0cf8cc4d78e47ef9debfb1e93a5c1197ca9a/docs/source/en/model_doc/bert.mdx?plain=1#L62) resources for an example (and in some cases, you can just copy it over)!

**Review by stevhliu — COMMENTED** [review]
Thanks for starting this; I left some comments on some things we're missing. 🤗
